SchedulerBot/SchedulerBot.Infrastructure.BotConnector/SchedulerBot.Infrastructure.BotConnector.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Bot.Connector" Version="4.0.0-alpha201802123" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\SchedulerBot.Business.Entities\SchedulerBot.Business.Entities.csproj" />
+    <ProjectReference Include="..\SchedulerBot.Database.Entities\SchedulerBot.Database.Entities.csproj" />
```

Comment:
We could remove this line since it's already a dependency of `SchedulerBot.Infrastructure.Interfaces.csproj`.